src/store/useDeveloperSettings.ts

```diff
@@ -1,32 +1,35 @@
 import { create } from 'zustand'
-import { persist } from 'zustand/middleware'
 
 interface DeveloperSettingsState {
+    
+    // Variables
     /**
      * Global mode
      */
-    developerMode: boolean;
+    debugMode: boolean;
+
+
+   // Actions
+   setDebugMode: () => void;
 }
 
 /**
  * All related to the App X Developer. Debugger Settings and more
  */
 const useDeveloperSettingsStore = create<DeveloperSettingsState>()(
-    // Save data in local storage
-    persist(
         (set, get) => {
             // Here we can add some logics
 
+            /**
+             * Toggles the debug mode.
+             */
+            const setDebugMode = () => set({debugMode: !get().debugMode}) 
```

Comment:
is `!get().debugMode` to get the reverse value of debugMode?

Reply:
Yes sir!  , is the quick way to get the opposite